solutions/swb-reference/src/backendAPI.ts

```diff
@@ -73,9 +73,12 @@ const apiRouteConfig: ApiRouteConfig = {
   userManagementService: new UserManagementService(
     new CognitoUserManagementPlugin(process.env.USER_POOL_ID!, aws)
   ),
-  costCenterService: new CostCenterService({
-    TABLE_NAME: process.env.STACK_NAME!
-  })
+  costCenterService: new CostCenterService(
+    {
+      TABLE_NAME: process.env.STACK_NAME!
```

Comment:
do we need the table name if we pass in the ddb service?

Reply:
Good question. It's being used by a preexisting function on this [line](https://github.com/aws-solutions/solution-spark-on-aws/blob/06ea0cf6117ff18176fd5f4bbeabba978833f0be/workbench-core/accounts/src/services/costCenterService.ts#L128)

Reply:
might be worth updating the signature to take an AccountService?

Reply:
Actually no that will probably spawn some merge conflicts with the feature branch.